src/plugins/ini/ini.c

```diff
@@ -23,17 +26,20 @@ int elektraIniClose(Plugin *handle, Key *parentKey);
 
 #include "contract.h"
 
+#define INTERNAL_ROOT_SECTION "GLOBALROOT"
```

Comment:
Why would you need the GLOBALROOT then? I would expect that
`kdb set user/inimountpoint/section1 i_am_a_global_key`
already writes into the global section?